R/bump-version.R

```diff
@@ -39,5 +39,12 @@ get_main_branch_remote <- function(remote) {
 
 get_main_branch_config <- function() {
   config <- gert::git_config()
-  config$value[config$name == "init.defaultbranch"]
+  init <- config[config$name == "init.defaultbranch",]
```

Comment:
It's a hacky workaround for demo projects, so maybe yes it'd be better to set up a remote? :grimacing:

Reply:
(I thought adding a remote would be too much complexity for now)

Reply:
Another workaround would be to make sure the GHA workflow files on Windows set the global git config to use main as init branch, before all the checks are done. Your call @krlmlr :-)

Reply:
Let's keep the workaround.